app/shared/telemetry/telemetry.py

```
@@ -0,0 +1,188 @@
+        """
+        self.service_name = service_name
+        self.service_version = service_version
+        self.enabled = enabled
+        self.tracer_provider: TracerProvider | None = None
+
+    def setup_telemetry(
+        self,
+        exporter_type: str = "console",
+        otlp_endpoint: str | None = None,
+        jaeger_endpoint: str | None = None,
+        sample_rate: float = 1.0,
```

Ruff (0.14.14)

[warning] Unused method argument: `sample_rate`

(ARG002)